### PR TITLE
docs(create-twitter-app): clarify twitter app permissions

### PR DESCRIPTION
**Pull request by vinckr**
Added a small section for Twitter app permissions.
It should probably be obvious, but I searched a while for my error until I realized I didnt give the app permission. 
Maybe this will help others to avoid this mistake.

### CODE REVIEW

File: docs/01-create-twitter-app.md [hunk 1]
```diff
@@ -91,6 +91,10 @@ https://github.com/gr2m/twitter-together
 
 This app will be used to create tweets that have previously been reviewed and accepted on our GitHub repository. It allows everyone to submit a tweet draft that we can discuss prior publishing.
 
+### Set app permissions
+
+When you have created your app, open `app settings` and set app permissions to "Read & Write".
+
 ## Save credentials
 
 "read and write" permissions are required. If you change the permission you must re-generate tokens for the change to take effect
```
Comment (gr2m) [inline docs/01-create-twitter-app.md:100]: it says it right the first sentence in the `Save credentials` section. Now it feels like repetition ... maybe we can rephrase the existing text to make it more clear?

**Review by gr2m — COMMENTED** [review]
Thanks a lot for trying to improve the documentation! I'm very sorry that it took a while for you to get right

**Comment by vinckr** [comment]
Hey, don't worry. This was mostly due to my inexperience with Twitter :). 

The action is working great and I think we will use it for automatically tweeting out new releases in the near future. 

I didnt see the section on the twitter perms, my bad! Should've just read the docs a little bit more attentively. 
I moved my stuff to the section, maybe it will help someone, but you have basically provided everything you need. 
Thanks alot :)

**Review by gr2m — APPROVED** [review]
Looks great, thank you!

**Pull request merged**